### PR TITLE
Add GM parachain with prefix 7013

### PR DESCRIPTION
**Pull request by arrudagates**
In the near future the GM chain will go for a Kusama slot, this PR is for requesting the prefix 7013 to be used with GM parachain account addresses. There is no hurry to merge this.

### CODE REVIEW

**Comment by arrudagates** [comment]
Just to give you more resources on GM, here's our github repo: https://github.com/ChaosDAO-org/gm_chain/

**Review by DanielCake-Baly — APPROVED** [review]

**Comment by arrudagates** [comment]
GM Parachain is currently in the lead of auction 45 on Kusama (https://kusama.subscan.io/auction/45). Can this be merged?

File: ss58-registry.json [hunk 1]
```diff
@@ -910,6 +910,15 @@
       "standardAccount": "*25519",
       "website": "https://tidefi.com"
     },
+    {
+      "prefix": 7013,
+      "network": "gm",
+      "displayName": "GM",
+      "symbols": ["FREN", "GM", "GN"],
+      "decimals": [12, 0, 0],
```
Comment (bkchr) [inline ss58-registry.json:918]: Why are these decimals 0?

Comment (arrudagates) [reply ss58-registry.json:918]: Because these tokens have no decimals, while for FREN 1 unit is 1000000000000, for GM and GN 1 unit is 1

**Pull request merged**